css-shapes-2/Overview.bs

```diff
@@ -147,13 +147,17 @@ The ''shape()'' Function</h4>
 	and inherits a number of limitations from SVG,
 	such as implicitly only allowing the ''px'' unit.
 
-	The ''shape()'' function uses a set of commands equivalent to the ones used by ''path()'',
+	The ''shape()'' function uses a set of commands roughly equivalent to the ones used by ''path()'',
 	but does so with more standard CSS syntax,
 	and allows the full range of CSS functionality,
 	such as additional units and math functions.
 	The commands used by ''shape()'' are dynamically turned into path segments when it is used for rendering,
 	e.g., when computing the rendered 'clip-path'.
 
+	In that sense, ''shape()'' is a superset of ''path()''. A ''path()'' can be easily converted to a ''shape()'',
+	but to convert a ''shape()'' back to a ''path()'' or to SVG requires information about the CSS environment (e.g.
+	current values of CSS custom properties), as well as the size of the [=reference box=] for computing percentages.
```

Comment:
There should be other CSS relative units included here, such as vw, cqw, and so on. Probably needs better wording?